src/citrine/informatics/design_spaces/hierarchical_design_space.py

```diff
@@ -150,6 +151,8 @@ class HierarchicalDesignSpace(EngineResource["HierarchicalDesignSpace"], DesignS
 
     """
 
+    _settings = properties.Optional(properties.Object(DesignSpaceSettings), "metadata.settings")
```

Comment:
These settings are just metadata intended to be carried to DS registration, not for users to tinker with. The only effect messing with them will have is screwing up filtering in the UI.